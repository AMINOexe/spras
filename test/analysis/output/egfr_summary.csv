name,number of nodes,number of edges,number of connected components,nodes in prize,nodes in sources,nodes in targets
tps-egfr-omicsintegrator1-params-3THRXWW_pathway.txt,26,20,6,26,1,25
tps-egfr-omicsintegrator1-params-5QH767V_pathway.txt,39,33,6,39,1,38
tps-egfr-omicsintegrator1-params-ITO5EQS_pathway.txt,12,8,4,12,0,12
tps-egfr-pathlinker-params-7S4SLU6_pathway.txt,12,15,1,7,1,6
tps-egfr-pathlinker-params-TCEMRS7_pathway.txt,23,27,1,14,1,13
