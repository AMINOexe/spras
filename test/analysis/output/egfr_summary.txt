Name	Number of nodes	Number of undirected edges	Number of connected components	Nodes in prize	Nodes in sources	Nodes in targets
test/analysis/input/egfr/tps-egfr-omicsintegrator1-params-3THRXWW_pathway.txt	26	20	6	26	1	25
test/analysis/input/egfr/tps-egfr-omicsintegrator1-params-5QH767V_pathway.txt	39	33	6	39	1	38
test/analysis/input/egfr/tps-egfr-omicsintegrator1-params-ITO5EQS_pathway.txt	12	8	4	12	0	12
test/analysis/input/egfr/tps-egfr-pathlinker-params-7S4SLU6_pathway.txt	12	15	1	7	1	6
test/analysis/input/egfr/tps-egfr-pathlinker-params-TCEMRS7_pathway.txt	23	27	1	14	1	13
