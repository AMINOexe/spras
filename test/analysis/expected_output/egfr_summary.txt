Name	Number of nodes	Number of edges	Number of connected components	Nodes in prize	Nodes in sources	Nodes in targets	Nodes in active	Parameter combination
input\egfr\tps-egfr-domino-params-V3X4RW7_pathway.txt	48	45	3	27	0	27	27	{slice_threshold: 0.3, module_threshold: 0.05}
input\egfr\tps-egfr-meo-params-GKEDDFZ_pathway.txt	1877	12845	1	621	1	620	621	{local_search: Yes, max_path_length: 3, rand_restarts: 10}
input\egfr\tps-egfr-omicsintegrator1-params-3THRXWW_pathway.txt	28	20	8	28	1	27	28	{b: 2, d: 10, g: 1e-3, r: 0.01, w: 0.1, mu: 0.008}
input\egfr\tps-egfr-omicsintegrator1-params-5QH767V_pathway.txt	39	31	8	39	1	38	39	{b: 10, d: 10, g: 1e-3, r: 0.01, w: 0.1, mu: 0.008}
input\egfr\tps-egfr-omicsintegrator1-params-ITO5EQS_pathway.txt	14	9	5	14	0	14	14	{b: 0.55, d: 10, g: 1e-3, r: 0.01, w: 0.1, mu: 0.008}
input\egfr\tps-egfr-omicsintegrator2-params-EHHWPMD_pathway.txt	593	591	2	531	1	530	531	{b: 2, g: 3}
input\egfr\tps-egfr-omicsintegrator2-params-IV3IPCJ_pathway.txt	704	702	2	616	1	615	616	{b: 4, g: 0}
input\egfr\tps-egfr-pathlinker-params-7S4SLU6_pathway.txt	14	17	1	6	1	5	6	{k: 10}
input\egfr\tps-egfr-pathlinker-params-TCEMRS7_pathway.txt	25	32	1	11	1	10	11	{k: 20}
